src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -2411,13 +2411,13 @@ void PolicyHandler::UpdateHMILevel(ApplicationSharedPtr app,
       LOG4CXX_INFO(logger_,
                    "Changing hmi level of application "
                        << app->app_id() << " to default hmi level " << level);
-      // Set application hmi level
-      application_manager_.ChangeAppsHMILevel(app->app_id(), level);
       // If hmi Level is full, it will be seted after ActivateApp response
       MessageHelper::SendHMIStatusNotification(
           app,
           mobile_apis::PredefinedWindows::DEFAULT_WINDOW,
           application_manager_);
+      // Set application hmi level
```

Comment:
Could you please explain why this change(moving the HMILevelChange to happen after sending the notification rather than before) was necessary?

Reply:
@ShobhitAd good point! These changes indeed seem to be unnecessary.
Reverted in a fixup commit 83c595c25a549134fb01b376313eb03dfaf185d0